rero_ils/modules/acq_accounts/api.py

```diff
@@ -56,6 +56,17 @@ class AcqAccount(IlsRecord):
     minter = acq_account_id_minter
     fetcher = acq_account_id_fetcher
     provider = AcqAccountProvider
+    pids_exist_test = {
```

Comment:
`pids_exist_test` can be renamed to something similar to `pids_exist_check`.. to avoid confusion with the real `tests` units testing.

Reply:
changed to: `pids_exist_check`